pwiz_tools/Skyline/TestFunctional/FullScanGraphTest.cs

```diff
@@ -97,6 +99,22 @@ protected override void DoTest()
             ClickFullScan(517, 1000);
             TestScale(516, 520, 0, 80);
 
+            //Check the rank and annotate functionality if we run in 
+            //onscreen mode
+            if (!Skyline.Program.SkylineOffscreen)

```

Comment:
Can this part of the test run even if the test is running offscreen?
We try to make the offscreen and onscreen versions of the test as similar as possible.

Reply:
In offscreen mode the annotation texts just do not exist, probably because Paint method is not invoked on the control and these annotations are added by during painting.

Reply:
Oh, that makes sense. Don't worry about this then.